src/server_manager/web_app/get_connected_app.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export class GetConnectedApp {
+  constructor(private dialog: polymer.Base) {
+    // Get connected is not a Polymer component, so we use `querySelector()` instead of `dialog.$`.
+    dialog.querySelector('#closeGetConnectedButton')
```

Comment:
`getConnectedDialog` is a `paper-dialog` and you're passing a `polymer.Base` object, so it's not clear to me why `dialog` is not a Polymer component.